packages/react-meteor-data/useTracker.js

```diff
@@ -73,12 +73,16 @@ function areHookInputsEqual(nextDeps, prevDeps) {
 
 let uniqueCounter = 0;
 
-function useTracker(reactiveFn, deps) {
+function useTracker(reactiveFn, deps, computationHandler) {
   const { current: refs } = useRef({});
 
   const [, forceUpdate] = useState();
 
   const dispose = () => {
+    if (refs.computationCleanup) {
```

Comment:
you should check for `typeof === 'function' or maybe even better, check the result of `computationHandler` and if !!value then check if for a function. If it is not a function console.warn it.

Reply:
I agree

Reply:
I added a check to the return value. I felt like checking the type of the initial function could be overkill, but maybe not. We do check the type of `deps`, but we don't check the type of `reactiveFn`. What do you think? I just don't want to bloat the size of of the package.